workspace_tools/targets.py

```diff
@@ -763,6 +763,8 @@ def __init__(self):
         Target.__init__(self)
         self.core = "Cortex-M0"
         self.extra_labels = ["NORDIC", "NRF51822_MKIT", "MCU_NRF51822", "MCU_NORDIC_16K"]
+        self.common_macros = ['NRF51']
```

Comment:
What does nrf51 mean? This can be done by expanding the base class (mcu nrf51) macro?
```
self.macros += new macros for each of the object derived from the nrf51 mcu.
```